src/common.jl

```diff
@@ -54,6 +60,7 @@ function SciMLBase.init(prob::LinearProblem, alg::Union{SciMLLinearSolveAlgorith
                         reltol=√eps(eltype(prob.A)),
                         maxiters=length(prob.b),
                         verbose=false,
+                        scale=one(eltype(prob.A)),
```

Comment:
You should use `Identity` when there's no predictioning to avoid overhead.  https://github.com/JuliaLinearAlgebra/IterativeSolvers.jl/blob/ae01dfe228138d0db9164eb343252150f6c0bfc6/src/common.jl#L28-L32